### ISSUE TITLE
ProjectMembers.remove throws exception

### ISSUE THREAD

**Issue by keydepth** [jdalrymple/gitbeaker#1852]
**Description**

I'm using `@gitbeaker/browser` latest version `29.2.4` 
and `Chrome ver: 91.0.4472.77(Official Build) (64 bit)`

Getting these errors when using `remove` method in `ProjectMembers`

```
TypeError: Failed to execute 'blob' on 'Response': body stream already read
    at index.js:formatted:9161
    at Generator.next (<anonymous>)
    at index.js:formatted:49
    at new Promise (<anonymous>)
    at __awaiter (index.js:formatted:26)
    at processBody (index.js:formatted:9159)
    at index.js:formatted:9373
    at Generator.next (<anonymous>)
    at fulfilled (index.js:formatted:29)
```

I checked in the expanded code index.js,
```
	return "application/json" === contentType ? 
		response.json().then((v=>v || {})) : 
		contentType.startsWith("text/") ? 
			response.text().then((t=>t || "")) : 
			response.blob().then((b=>Buffer.from(b)))
```

I checked in the source code, But I did not understand for this error.

https://github.com/jdalrymple/gitbeaker/blob/f866037c6f7c975024acb55c7b16fa975470a351/packages/gitbeaker-browser/src/KyRequester.ts#L54

**Steps to reproduce**

Run this code.

```
    const GitlabApi = new Gitlab({
        host: 'https://gitlab.com',
        token: 'XXXX',
        requestTimeout: 15000
    });
    await GitlabApi.ProjectMembers.remove(project_id, user_id).catch(async err => {
        // error
        console.log(err);
        alert(err);
        return null;
    });
```

**Expected behaviour**

Does not reach in cache() path .

**Actual behaviour**

cache() path reached.

**Possible fixes**

Fix stream read timing for `body stream already read`, maybe.

**Comment by jdalrymple** [comment]
Ill give it a look!

**Comment by jdalrymple** [comment]
Not really sure why this would happen. Its only being read once :thinking:

**Comment by jdalrymple** [comment]
Does this still happen with > 30? Could be related to the build problems?

**Comment by keydepth** [comment]
OK. Ill check with > 30

**Comment by keydepth** [comment]
I'm using `@gitbeaker/browser` latest version **`30.2.0`** 
and `Chrome ver: 91.0.4472.77(Official Build) (64 bit)`

Getting these errors when using `remove` method in `ProjectMembers`
It seems changed.

```
ReferenceError: Buffer is not defined
    at KyRequester.ts:46
```

I checked in the expanded code js,

KyRequester.ts
```
  return response.blob().then((b) => Buffer.from(b as unknown as SharedArrayBuffer));
```

buffer-es6.js
```
/**
 * Functionally equivalent to Buffer(arg, encoding) but throws a TypeError
 * if value is a number.
 * Buffer.from(str[, encoding])
 * Buffer.from(array)
 * Buffer.from(buffer)
 * Buffer.from(arrayBuffer[, byteOffset[, length]])
 **/
Buffer.from = function (value, encodingOrOffset, length) {
  return from(null, value, encodingOrOffset, length)
};
```

**Comment by keydepth** [comment]
I watched value of `b`.
`Buffer.from(b as unknown as SharedArrayBuffer)` in this timing `b` is 
```
Blob {size: 0, type: ""}
```

**Comment by keydepth** [comment]
@jdalrymple 
Thank you for your comment.
`Buffer` module was not `import`.
So, In browser, Is it not need `import Buffer`?

**Comment by jdalrymple** [comment]
Weird, it should be included with the dom inclusion during the build. Ill give it another look! Thank you for the deep dive

**Comment by jdalrymple** [comment]
:rocket: Issue was released in [`34.0.0`](https://github.com/jdalrymple/gitbeaker/releases/tag/34.0.0) :rocket:
